ts/nni_manager/training_service/reusable/shared_storages/azureblobStorageService.ts

```diff
@@ -206,4 +216,21 @@ export class AzureBlobSharedStorageService extends SharedStorageService {
             return Promise.reject(errorMessage);
         }
     }
+
+    public async cleanUp(): Promise<void> {
+        if (this.localMounted !== 'nnimount') {
+            return Promise.resolve();
+        }
+        try {
+            const result = await cpp.exec(`sudo umount -l ${this.localMountPoint}`);
+            if (result.stderr) {
+                throw new Error(result.stderr);
+            }
+        } catch (error) {
+            const errorMessage: string = `${this.storageType} Shared Storage: get account key failed, error is ${error}`;
```

Comment:
why the message contains `get account key`?

Reply:
it is a mistake, thank you, fix it